api/scripts/certification/next-gen/import-pilot-certification-centers-from-csv.js

```diff
@@ -37,16 +37,16 @@ function buildCertificationCentersPilotsList({ featureId, certificationCentersId
   });
 }
 
-async function hasNoV3CertificationCenters({ certificationCentersIds }) {
+async function onlyV3CertificationCenters({ certificationCentersIds }) {
   const v3CertificationIds = await knex
```

Comment:
renommer ici aussi :)

```suggestion
const v2CertificationIds = await knex
```

Reply:
Done